src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/LanternApp.java

```diff
@@ -5,11 +5,20 @@
 import com.crashlytics.android.Crashlytics;
 import io.fabric.sdk.android.Fabric;
 
+import org.getlantern.lantern.model.SessionManager;
+
 public class LanternApp extends Application {
     private static final String TAG = "LanternApp";
+    private static SessionManager session;
 
     @Override
     public void onCreate() {
```

Comment:
Hey not related to this PR @atavism but seems like we should be calling `super.onCreate()` here technically.